x-pack/plugins/security_solution/public/common/links/links.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AppDeepLink, AppNavLinkStatus, Capabilities } from '@kbn/core/public';
+import { LicenseType } from '@kbn/licensing-plugin/common/types';
+import { get } from 'lodash';
+import { ExperimentalFeatures } from '../../../common/experimental_features';
+import { appLinks } from './structure';
+import { LinkItem, Feature } from './types';
+interface LinkProps {
+  enableExperimental: ExperimentalFeatures;
+  isBasic: boolean;
+  capabilities?: Capabilities;
+}
+
+const createDeepLink = (link: LinkItem, linkProps?: LinkProps): AppDeepLink => ({
+  ...(link.items && link.items.length ? { deepLinks: reduceDeepLinks(link.items, linkProps) } : {}),
+  ...(link.icon != null ? { euiIconType: link.icon } : {}),
+  ...(link.image != null ? { icon: link.image } : {}),
```

Comment:
🤔

Reply:
are you sure, it was in our doc

Reply:
I believe this `image` is the section "screenshot" image used by Pablo's group landing pages. And the `icon` is the one used there as well.
We don't have any kind of icon or image in the deepLinks right now, I don't know what they would do, I think we should stick to the same deepLinks values as we have now.
Maybe we should name the `icon` and `image` differently to not get confused?